bundles/org.openhab.binding.metofficedatahub/README.md

```diff
@@ -167,7 +167,7 @@ current-forecast-**plus02**#air-temp-current
 ### Configuration (*.things)
 
 ```java
-Bridge metofficedatahub:site:metoffice [siteRateDailyLimit=200, siteApiKey="<Site Specific API Key>"] {
+Bridge metofficedatahub:account:metoffice [siteRateDailyLimit=200, siteApiKey="<Site Specific API Key>"] {
```

Comment:
Can you also fix lines 29 and 34 which still mentions `bridge` rather than `account`?